metacat-common/src/main/java/com/netflix/metacat/common/dto/notifications/sns/SNSMessageType.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.metacat.common.dto.notifications.sns;
+
+/**
+ * Enumeration of the various types of SNS events there can be.
+ *
+ * @author tgianos
+ * @since 0.1.47
+ */
+public enum SNSMessageType {
+    TABLE_CREATE,
+    TABLE_DELETE,
+    TABLE_UPDATE,
+    TABLE_PARTITIONS_UPDATE,
```

Comment:
Is this a rhetorical question? The event that comes in from internal has a qname and only one qname for the entire payload of partitions so I assume that is the qname of the table they're for... no? If that's not the case we probably need a whole other big feature outside the scope of this PR to fix that before we can send the notification of a table changing. Unless you're saying we need to parse each of the partitions to figure out what table they belong to?